samples/net/wifi/boards/rd_rw612_bga.conf

```diff
@@ -87,7 +87,7 @@ CONFIG_MBEDTLS_USER_CONFIG_FILE="wpa_supp_els_pkc_mbedtls_config.h"
 CONFIG_ENTROPY_GENERATOR=y
 CONFIG_MBEDTLS_PSA_CRYPTO_C=y
 CONFIG_MBEDTLS_ENTROPY_C=y
-CONFIG_MBEDTLS_ZEPHYR_ENTROPY=y
+CONFIG_MBEDTLS_ENTROPY_POLL_ZEPHYR=y
```

Comment:
I think it is best to queue this as a hotfix in a separate PR if it is not already. (CC @valeriosetti )

Reply:
This was fixed in https://github.com/zephyrproject-rtos/zephyr/pull/77517.
@jilaypandya thanks for the fix!